layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+{% include "template-variables" layout_product: true %}
+
+<html class="{{ view_mode }} {{ language_flags_mode }} {{ language_names_mode }} {{ language_menu_mode }}" lang="{{ page.language_code }}" data-view-state="{{ view_mode }}">
+  <head prefix="og: http://ogp.me/ns#">
+    {% assign item_list_page = true %}
+    {% include "template-variables" %}
+    {% include "html-head" sidebar: true %}
+    {% include "template-styles" %}
+    {% include "common-page-variables" %}
+  </head>
+
+  {%- capture bottom_content_html %}
+    {% unless editmode %}
+      {% content bind=product name="content" %}
+    {% endunless %}
+  {% endcapture %}
+
+  {% assign bottom_content_size = bottom_content_html | strip | size %}
+
+  {% capture gallery_content_html %}
+    {% unless editmode %}
+      {% content bind=product name="gallery" %}
+    {% endunless %}
+  {% endcapture %}
+
+  {% assign gallery_content_size = gallery_content_html | strip | size %}
+
+  <body class="{% if sidebar_active %}common-page sidebar-active{% else %}sidebar-inactive{% endif %} item-page{% if site.data.has_header_bg_color %} header-top-with-bg{% endif %}">
+    {% include "template-svg-spritesheet" %}
+
+    <div class="site-container">
+      {% include "site-header" %}
+      {% include "common-page-variables" %}
+
+      <div class="page-body js-bg-picker-area">
+        <div class="js-background-type {{ body_bg_type }}">
+          <div class="background-color js-background-color"></div>
+
+          <div {% if sidebar_active %} class="sidebar-active"{% endif %}>
+            {% if sidebar_active %}
+              {% include "site-sidebar" %}
+            {% endif %}
+
+            <main class="page-content" role="main">
+              <div class="main-inner-row content-full">
+                <div class="main-content top-row">
+                  <div class="wrap">
+                    <div class="inner product-content">
+                      {% include "menu-breadcrumbs-sd" %}
+                      <div class="items-body" {{ edy_intro_edit_text }}>
+                        <div class="flex-col">
+                          <div class="content-illustrations">
+
+                            {% if product.image != blank %}
+                              {% assign item_image_state = "with-image" %}
+                            {% else %}
+                              {% assign item_image_state = "without-image" %}
+                            {% endif %}
+
+                            <div class="content-item-box {{ item_image_state }} js-content-item-box">
+                              <div class="item-top">
+                                {%- if product.image != blank -%}
+                                  <div class="top-inner aspect-ratio-inner product-page product-image">
```

Comment:
`product-image` class is visible only when product already has an image, meaning image area is clickable only when image is visible. The image area should be clickable either way.
```suggestion
                              <div class="item-top product-image">
                                {%- if product.image != blank -%}
                                  <div class="top-inner aspect-ratio-inner product-page">
```